dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/DotTemplateTool.java

```diff
@@ -39,15 +34,34 @@
  */
 public class DotTemplateTool implements ViewTool {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a constructor to the class, or provide default values. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1258)